content/terminal/openbb-copilot.md

```diff
@@ -5,215 +5,340 @@ description: Learn how to use OpenBB Copilot to interact with the OpenBB Termina
 <div style={{display: 'flex', justifyContent: 'center'}}>
-<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/news_openbb_api_chat_example.jpg" alt="openbb copilot api" width="50%" height="50%" />
+<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/direct_retrieval_example.png" alt="direct retrieval example" width="50%" height="50%" />
 </div>
 
-To query a custom endpoint, please explore the [Custom Backend](/terminal/custom-backend) documentation.
+If OpenBB Copilot retrieved data directly from a data source, you can add that
+data to your dashboard by hovering over the appropriate citation, and clicking
+on the "Add to dashboard" button:
+
+<div style={{display: 'flex', justifyContent: 'center'}}>
+<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/add_to_dashboard_from_citation.png" alt="add to dashboard from citation" width="70%" height="70%" />
+</div>
 
-### Searching the Web
+## Producing charts and tables using OpenBB Copilot
+OpenBB Copilot can produce charts and tables using the data that it has access
+to. We call these special outputs "artifacts".
 
-The OpenBB Copilot has the ability to search the web for information that is not available in the data sources it has access to. This allows you to ask questions that go beyond the data displayed on your dashboard.
+For charts, OpenBB Copilot can currently produce the following plot types:
+- Line chart
+- Vertical bar chart
+- Scatter plot
 
-To search the web, the user needs to be explicit in their query by adding the `@web` keyword in the query.
+To produce a chart or a table, mention the output that you want in your query.
+For example, to produce a chart of the latest stock price for Apple, you can
+ask OpenBB Copilot to "Create a chart of the latest stock price for Apple":
 
 <div style={{display: 'flex', justifyContent: 'center'}}>
-<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/web_search_chat_example.jpg" alt="web search openbb copilot" width="80%" height="80%" />
+<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/chart_artifact.png" alt="chart artifact openbb copilot" width="60%" height="60%" />
 </div>
 
-## Citations, Status Updates, and Artifacts
+If you want a table instead, you can just modify your query slightly to ask for
+a table:
 
-When querying OpenBB Copilot, it is important to understand which data OpenBB
-Copilot used to answer your query. As a result, OpenBB Copilot will always cite the data
-source it referenced when responding to your query.
+<div style={{display: 'flex', justifyContent: 'center'}}>
+<img src="https://openbb-assets.s3.amazonaws.com/docs/copilot/table_artifact.png" alt="table artifact openbb copilot" width="60%" height="60%" />
+</div>
+
+This behaviour works across all structured data.
+
+### Converting artifacts to widgets
+Every artifact that OpenBB Copilot generates can be converted to a widget. To do
+this, click the "Create widget from text" or "Create widget from table" button:
```

Comment:
Here the screenshot actually says "Create widget from chart", so we can be more general like:

"...click the "Create widget from [text, table or chart]" button:"

Reply:
Good catch!